docs/release-notes/2.43.0.md

```diff
@@ -0,0 +1,210 @@
+
+This new functionality is enabled through a dedicated PBM collector within the MongoDB exporter, allowing you to easily monitor your PBM setup with the following metrics:
+
+- `mongodb_pbm_cluster_backup_configured`
+- `mongodb_pbm_agent_status`
+- `mongodb_pbm_cluster_pitr_backup_enabled`
+- `mongodb_pbm_backup_size`
+- `mongodb_pbm_backup_duration_seconds`
+
+These metrics are enabled by default. If you’re already using PBM, you’ll automatically start receiving these metrics without additional configuration.
+
+### Improved troubleshooting
+
+We've enhanced PMM's troubleshooting capabilities to provide you with better insights and more efficient problem-solving tools:
+
+#### Enhanced PMM Server logs
+
+The default number of log lines for each log file returned by `httsp://<pmm-server>/logs.zip` endpoint has been increased from 1,000 to 50,000.
+
+Additionally, the endpoint now includes a customizable line-count parameter in the download URL, allowing you to specify a custom number of log lines or opt for unlimited log size. For more information, see the [API documentation](https://percona-pmm.readme.io/reference/logs).
+
+#### Streamlined Kubernetes diagnostics 
+
+New PMM Client Docker images now include essential troubleshooting tools:
+
+- `tar` enables easier file transfer in and out of containers using the `kubectl cp` command.
+- `curl` allows direct checking of exporters to verify their proper functioning.
+
+### Improved security
+
+This update addresses several security issues related to the GNU C Library (Glibc), specifically affecting the Name Service Cache Daemon (nscd). Additionally, it resolves multiple security vulnerabilities by upgrading various third-party packages.
+
+### Important changes to PMM Update process
+
+To ensure you can access the most up-to-date and secure versions, we've implemented two key changes to the update process:
+
+#### Transition to a dedicated repository
+
+Since July 1st 2024, PMM updates are distributed exclusively through the dedicated `repo.percona.com/pmm2-client` repository. 
+
+To smoothly transition to this repository and verify that your system is correctly configured, download and run our automated script from GitHub: [check_percona_packages.py](https://raw.githubusercontent.com/Percona-Lab/release-aux/main/scripts/check_percona_packages.py).
+
+For detailed instructions and more information about this change, check out [our recent blog post](https://www.percona.com/blog/ensure-the-correct-repositories-are-enabled-for-percona-packages/) and [updated installation instructions](../setting-up/index.md).
+
+#### Update process for older PMM versions
+
+With the migration of PMM’s base image from CentOS 7 to Oracle Linux 9 last year, users running PMM version 2.38 or earlier may experience issues when updating PMM to the latest version using the UI **Update** button.
```

Comment:
isn't this because of the discontinuance of the CentOS7 repos?  we should say that if so.  RedHat is the reason why...not something we did.